run.c

```diff
@@ -324,14 +324,14 @@ void compute_accelerations(int is_PM)
             energy_statistics();
     }
 
-    grav_short_tree();		/* computes gravity accel. */
-
-    if(All.Ti_Current == 0)
-        grav_short_tree();		/* For the first timestep, we redo it
-                             * to allow usage of relative opening
-                             * criterion for consistent accuracy.
-                             */
+    /* For the first timestep, we do tree force twice
+     * to allow usage of relative opening
+     * criterion for consistent accuracy.
+     */
+    if(FirstStep)
```

Comment:
We can probably move the first short_tree calculation to init?

Reply:
Alas, no - the OldAcc calculation uses GravPM, so that needs to be computed before this.

Reply:
Perhaps setting OldAcc to the snapshot numbers, then this can be avoided?

Reply:
Looks like it is not worth it to go that far to avoid this flag. We have to do something special here anyways. But oldacc can be lifted out from P -- it can be computed before the tree walk in a local array from GravPM and GravAccel.

Reply:
You mean saving and reloading OldAcc with a snapshot save? I did implement that, but once I had done so it seemed not really worth the effort (although it makes startup a bit faster).

I will take a look at lifting OldAcc in another pr - it doesn't really fit here.